src/main/java/org/folio/clients/InventoryClient.java

```diff
@@ -145,20 +175,26 @@
     return map;
   }
 
-  public Optional<JsonObject> getHoldingsByInstanceId(String instanceID, OkapiConnectionParams params) {
+  public Optional<JsonObject> getHoldingsByInstanceId(String instanceID, String jobExecutionId, OkapiConnectionParams params) {
     String endpoint = buildQueryEndpoint(resourcesPathWithPrefix(HOLDING) + QUERY_LIMIT_PATTERN + HOLDINGS_LIMIT,
         params.getOkapiUrl(), String.format(QUERY_PATTERN_HOLDING, instanceID));
-    return getRequest(params, endpoint);
-  }
-
-  public Optional<JsonObject> getItemsByHoldingIds(List<String> holdingIds, OkapiConnectionParams params) {
-    return ClientUtil.getByIds(holdingIds, params, resourcesPathWithPrefix(ITEM) + QUERY_LIMIT_PATTERN + HOLDINGS_LIMIT,
-        QUERY_PATTERN_ITEM);
-  }
-
-  public Map<String, JsonObject> getContributorNameTypes(OkapiConnectionParams params) {
-    String endpoint = resourcesPathWithPrefix(CONTRIBUTOR_NAME_TYPES) + LIMIT_PARAMETER + REFERENCE_DATA_LIMIT;
-    return getReferenceDataByUrl(endpoint, params, CONTRIBUTOR_NAME_TYPES);
+    try {
+      return getRequest(params, endpoint);
+    } catch (ClientException exception) {
+      errorLogService.saveGeneralError("Error while getting holdings by instance ids " + exception.getMessage(), jobExecutionId, params.getTenantId());
```

Comment:
Log the instance Id, so that we know for which instance we couldn't get the holdings

Reply:
ok nvm, the details should be there in the exception message thrown?

Reply:
will add